packages/client/src/comp/states/StateForm.tsx

```diff
@@ -98,9 +98,12 @@ export const StateForm: React.FC<Props> = ({ignoreError}) => {
     throw Error(`Locale state ${locale.state} is not implemented`)
   }
 
+  const inDeadline = inPrimaryDeadline(address.state as State)
```

Comment:
`locale.State` is always a `State`

Reply:
I don't think it is, maybe we should change `address.ts`?
https://github.com/vote-by-mail/website/blob/4c573208cae9226e2c221e7023a9d3f20c3e4378/packages/common/address.ts#L9-L11

Reply:
I think you're wrong.  See this: https://github.com/vote-by-mail/website/pull/80/commits/d10f1ffffe0ca6bd4a1ea02d01e25548b8f62a40 where I use `locale.State` and do not have to cast.